src/client/app/needs/search_completed_needs.html

```diff
@@ -0,0 +1,82 @@
+<section>
+    <div class="contentWrapper">
+        <div class="row">
+            <div class="col-xs-12 col-sm-12 col-md-9 col-lg-8 col-md-offset-2 col-lg-offset-2">
+                <div class="btn-group col-xs-12 col-sm-4 col-md-4 col-lg-4 marginBottom" dropdown>
+                    <button class="btn btn-info dropdown-toggle col-xs-12" type="button" id="dropdownMenu2" dropdown-toggle aria-haspopup="true" aria-expanded="true">
+                        Категорії&nbsp;
+                        <span class="caret"></span>
+                    </button>
+                    <ul class="dropdown-menu col-xs-12" aria-labelledby="dropdownMenu2">
+                        <li><a href="#">Stuff</a></li>
+                        <li><a href="#">Stuff</a></li>
+                        <li><a href="#">Stuff</a></li>
+                        <li role="separator" class="divider"></li>
+                        <li><a href="#">Stuff from different category</a></li>
+                        <li><a href="#">Stuff from different category</a></li>
+                    </ul>
+                </div>
+                <location-filter-directive locations="vm.data" setlocation="vm.setlocation"></location-filter-directive>
```

Comment:
this will be changed as well when you rename the directive, but again, I wouldn't use a `filter` word here, it's confusing. Think about another name.